src/main/java/org/myrobotlab/arduino/Msg.java

```diff
@@ -2,18 +2,16 @@
 
 
 import java.io.ByteArrayOutputStream;
-import java.io.OutputStream;
+import java.io.FileOutputStream;

```

Comment:
This file is generated.. was the template updated?

Reply:
I tried to do organize import based on a package level, but it didn't work.
No the file wasn't regenerated, it doesn't use the import, so "organized" means it goes away.
Next time its generated it will re-create the not used OutputStream reference.

Reply:
ok.. but this file shouldn't be modified unless it's generated...

Reply:
its been reverted to its original generated, yet unused java-lint warning import state